### ISSUE TITLE
Copyright information in the <module>.psd1 file

### ISSUE THREAD

**Issue by johlju** [PowerShell/DscResources#289]
When I look at the xSQLServer.psd1 file in xSQLServer I see that the copyright information has the year of 2014. Must it be so, or should this be changed to the current year when a release is done? If it doesn't matter what year it is, could we changed it to the current year just so the module feel more fresh? 😄 

```
# Copyright statement for this module
Copyright = '(c) 2014 Microsoft Corporation. All rights reserved.'
```

@zjalexander Maybe this one is something you can answer?

**Comment by zjalexander** [comment]
investigating

**Comment by zjalexander** [comment]
yes, update it to the current year :)

**Comment by johlju** [comment]
Great I will do that! 
Should we add this to the Maintainers.md that this should be update continuously each year?

**Comment by zjalexander** [comment]
Sure, you wanna do a quick PR and i'll review?

**Comment by johlju** [comment]
I will put together a PR and tag you when it's ready for review

**Comment by johlju** [comment]
This was merged in #293. This issue did not auto-close for some reason.
